.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         julia-version:
-          - "1.0"
+          - "1.6"
```

Comment:
Minimum required version is 1.0 though https://github.com/staticfloat/ObjectFile.jl/blob/c1c705c3eb2077b122cba6193b95523c2f2a23da/Project.toml#L14

Reply:
I'm going to ask Tim to bump that in his PR.